webapp/views.py

```diff
@@ -313,3 +316,79 @@ def notices():
             total_results=total_results,
         ),
     )
+
+
+# USN API
```

Comment:
It feels that this could be in it's own file (`webapp/security/views.py` ?) 
I think it could help splitting a bit the different views that provides templates, and the others that are APIs.

Reply:
Yeah I wrote that in the description of the PR, with ubuntu.com growing so much it feels like we need to split things up.

Reply:
I agree, but also is it fine to define the database schema in the view?

Reply:
No, it should live in it's own file in my opinion

Reply:
Ah okay. So it is here provisionally or something?

Reply:
Yeah, until we decide if it's ok to split this into it's own module